src/login/login.component.tsx

```diff
@@ -26,7 +26,7 @@ export default function Login(props: LoginProps) {
         authResult => {
           setCheckingIfLogged(false);
           if (authResult.authenticated) {
-            props.history.push("/home");
+            navigate(props, config.links.loginSuccess);
```

Comment:
A good login page is one that will send you to the URL you were attempting to go to before you were redirected to the login page.

For example, if you go to https://openmrs-spa.org/openmrs/spa/patient/8673ee4f-e2ab-4077-ba55-4980f408773e/chart and you're not logged in, you should be sent to the login page and then sent back to that page when you are done. This is the most common reason people end up on the login page.

I would love to support that use case, in addition to what you have here.

Reply:
Oh yeah, absolutely! Filed https://issues.openmrs.org/browse/MF-103